src/desktop_develop.ts

```diff
@@ -329,37 +368,33 @@ class DesktopDevelopBuilder {
         );
     }
 
-    async build(type, buildVersion) {
-        if (type.startsWith('win')) {
+    private async build(type: Type, buildVersion: string): Promise<void> {
+        if (isWindows(type)) {
             return this.buildWin(type, buildVersion);
         } else {
             return this.buildLocal(type, buildVersion);
         }
     }
 
-    async buildLocal(type, buildVersion) {
+    private async buildLocal(type: Type, buildVersion: string): Promise<void> {
         await fsProm.mkdir('builds', { recursive: true });
         const repoDir = path.join('builds', 'element-desktop-' + type + '-' + buildVersion);
-        await new Promise((resolve, reject) => {
-            rimraf(repoDir, (err) => {
-                err ? reject(err) : resolve();
-            });
-        });
+        await rm(repoDir);
         logger.info("Cloning element-desktop into " + repoDir);
         const repo = new GitRepo(repoDir);
         await repo.clone(DESKTOP_GIT_REPO, repoDir);
         logger.info("...checked out 'develop' branch, starting build for " + type);
 
         await this.writeElectronBuilderConfigFile(type, repoDir, buildVersion);
-        if (type == 'linux') {
+        if (type === Type.Linux) {
             await setDebVersion(
                 buildVersion,
                 path.join(repoDir, 'element.io', 'nightly', 'control.template'),
                 path.join(repoDir, 'debcontrol'),
             );
         }
 
-        let runner;
+        let runner: IRunner;
```

Comment:
github won't let me get at it, but the case tree in the switch below should use the new type enum